src/operator-docs/filtering/skipWhile.ts

```diff
@@ -1,6 +1,44 @@
 import { OperatorDoc } from '../operator.model';
 
 export const skipWhile: OperatorDoc = {
-  'name': 'skipWhile',
-  'operatorType': 'filtering'
+  name: 'skipWhile',
+  operatorType: 'filtering',
+  signature: `
+    public skipWhile(predicate: Function): Observable<T>
```

Comment:
Do we stick to Function as signature type or should we use arrow function to provide the return type? Example for this on could be: `public skipWhile(predicate: () => boolean): Observable<T>`

Reply:
I think we also decided not to include generics in signatures, so we can remove `<T>`.